src/lib/components/lines-and-dots/event-type-filter.svelte

```diff
@@ -46,32 +57,79 @@
       ? $eventTypeFilter.filter((type) => type !== value)
       : [...$eventTypeFilter, value];
   };
+
+  $: filterActive = $eventTypeFilter.length < defaultOptions.length;
 </script>
 
-<div class="flex flex-wrap items-center items-center justify-center gap-4">
-  {#each options as option}
-    <Tooltip width={200} text={option.tooltip} topRight>
-      <div class="flex items-center">
+<MenuContainer {open}>
+  <MenuButton controls="status-menu">
+    <div
+      slot="leading"
+      class="flex h-6 w-6 flex-col items-center justify-center rounded-full transition-colors duration-200"
+      class:bg-interactive={filterActive}
+    >
+      <Icon name="filter" class={filterActive && 'pt-0.5 text-white'} />
+    </div>
+    {translate('events.event-types')}
+  </MenuButton>
+  <Menu
+    id="event-type-menu"
+    keepOpen
+    position="top-right"
+    class="w-[240px] md:w-[400px]"
+  >
+    <MenuItem
+      data-testid={translate('common.all')}
+      on:click={() => {
+        $eventTypeFilter = defaultOptions;
+      }}
+    >
+      <Checkbox
+        on:change={() => {
+          $eventTypeFilter = defaultOptions;
+        }}
+        slot="leading"
+        checked={$eventTypeFilter.length === defaultOptions.length}
+        label={translate('common.all')}
+        labelHidden
+      />
+      {translate('common.all')}
+    </MenuItem>
+    <MenuItem
+      data-testid={translate('common.none')}
+      on:click={() => {
+        $eventTypeFilter = [];
+      }}
+    >
+      <Checkbox
+        on:change={() => {
+          $eventTypeFilter = [];
+        }}
+        slot="leading"
+        checked={!$eventTypeFilter.length}
+        label={translate('common.none')}
+        labelHidden
+      />
+      {translate('common.none')}
+    </MenuItem>
+    <MenuDivider />
+    {#each options as option}
+      <MenuItem
+        data-testid={option.label}
```

Comment:
Wondering if we want to align items to start instead of center here (or in the `MenuItem` component itself) if there is a description ⬇️ 
```suggestion
        class="!items-start"
        data-testid={option.label}
```

| Before | After |
|-|-|
|![Screenshot 2024-09-24 at 11 02 53 AM](https://github.com/user-attachments/assets/93e49fa9-902a-4ce9-8a5c-2973d525ebd9)|![Screenshot 2024-09-24 at 11 03 05 AM](https://github.com/user-attachments/assets/b065e3d4-e312-4a35-865f-3b3c3491a4ef)|

Reply:
Hmm yeah I could be convinced either way

Reply:
But should do it MenuItem if we want to do it